### ISSUE TITLE
Deploy as a Nuget package on nuget.org / chocolatey.org

### ISSUE THREAD

**Issue by jairbubbles** [hillin/FASTBuilder#2]
It would be really cool to deploy the tool as a NuGet package on chocolatey.org or nuget.org (or both!)

If you don't know much about it I can help 😄

**Comment by hillin** [comment]
Yes it's definitely doable when the program gets stable, although I always have the prejudice that NuGet is just somewhere you can get libs :)

**Comment by jairbubbles** [comment]
There are a few applications on nuget.org too but clearly it's better on chocolatey.org.

You just have to package with the .exe in `tools` and it will automatically install when doing
```
choco install fastbuild-dashboard
```
